test/paper-item.html

```diff
@@ -91,6 +99,37 @@
           }, 1);
         });
 
+        test('enter triggers a click event', function(done) {
+          MockInteractions.pressEnter(item);
+          Polymer.Base.async(function(){
+            // You need two ticks, one for the MockInteractions event, and one
+            // for the button event.
+            Polymer.Base.async(function(){
+              expect(clickHandler.callCount).to.be.equal(1);
+              done();
+            }, 1);
+          }, 1);
+        });
+
+        test('click triggers a click event', function(done) {
+          MockInteractions.tap(item);
```

Comment:
This test seems to be testing `MockInteractions.tap`, not `item`. `MockInteractions.tap` always causes a `click` to be fired on the element it is used on. So, all this test does is verify that that happens.